src/app/fdctl/monitor/monitor.c

```diff
@@ -350,6 +357,8 @@ run_monitor( config_t * const config,
       PRINT( " | " ); printf_stale   ( &buf, &buf_sz, (long)(0.5+ns_per_tic*(double)(toc - cur->cnc_heartbeat)), 1e8 /* 100 millis */ );
       PRINT( " | " ); printf_heart   ( &buf, &buf_sz, cur->cnc_heartbeat, prv->cnc_heartbeat        );
       PRINT( " | " ); printf_sig     ( &buf, &buf_sz, cur->cnc_signal,    prv->cnc_signal           );
+      PRINT( " | " ); printf_rate    ( &buf, &buf_sz, 1e9, 0., cur->nivcsw, prv->nivcsw, dt );
+      PRINT( " | " ); printf_rate    ( &buf, &buf_sz, 1e9, 0., cur->nvcsw,  prv->nvcsw,  dt );
```

Comment:
Can you align it better? Probably add spaces to the 2 previous and following lines to correspond to the 1e9 and 0